tests/stress_tests/scripts/get_testdata.py

```diff
@@ -155,9 +149,21 @@ def main():
                                       models_dir=args.omz_models_out_dir,
                                       cache_dir=args.omz_cache_dir,
                                       jobs_num=DOWNLOADER_JOBS_NUM)
+    models_available = subprocess.check_output(cmd + " --print_all", shell=True, universal_newlines=True).split("\n")
```

Comment:
Please route all calls through run wrapper. Consider unifying it with run from run_memcheck.py so that you can get stdout

Reply:
Let's do this in another PR to unblock fix